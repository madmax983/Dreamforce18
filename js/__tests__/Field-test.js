import React from 'react';
import Field from '../Field.js';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<Field />).toJSON();
    expect(tree).toMatchSnapshot();
});