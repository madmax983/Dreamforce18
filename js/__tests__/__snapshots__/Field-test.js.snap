// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Array [
        Object {},
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontSize": 15,
              "fontWeight": "bold",
              "marginBottom": 1,
              "marginLeft": 20,
              "marginRight": 20,
              "marginTop": 15,
            },
            undefined,
            undefined,
          ],
        ]
      }
    />
  </View>
  <View
    collapsable={undefined}
    style={
      Object {
        "borderBottomColor": "#bdc6cf",
        "borderBottomWidth": 1,
        "marginLeft": 20,
        "marginRight": 20,
        "transform": Array [
          Object {
            "translateX": 0,
          },
        ],
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={undefined}
      style={
        Array [
          Object {
            "color": "#86939e",
            "minHeight": 36,
            "width": 750,
          },
          Object {
            "fontSize": 17.5,
          },
          undefined,
        ]
      }
      value={undefined}
    />
  </View>
</View>
`;
